CITATION.cff

```diff
@@ -0,0 +1,37 @@
+abstract: "marimo is a reactive notebook for Python — runs reproducible experiments, executes as a script, deploys as an app, and versions with git."
+authors:
+  - family-names: Agrawal
+    given-names: Akshay
+    orcid: "https://orcid.org/<insert-number>"
+  - family-names: Scolnick
+    given-names: Myles
+    orcid: "https://orcid.org/<insert-number>"
+cff-version: 1.2.0
+date-released: "2023-08-15"
+identifiers:
```

Comment:
Decide on whether to include the identifiers section. If yes, please provide the appropriate DOIs, Software Heritage numbers, or URLs. Examples of valid identifiers include:
DOI: 10.5281/zenodo.1003149 (The concept DOI of the work)
SWH: swh:1:dir:bc286860f423ea7ced246ba7458eef4b4541cf2d (The Software Heritage identifier for a specific version)
URL: https://github.com/citation-file-format/citation-file-format/releases/tag/1.1.0 (The GitHub release URL of a specific tag)
OTHER: arXiv:2103.06681 (The ArXiv preprint of a related paper)

Reply:
if this is optional - i am fine to remove it

Reply:
> if this is optional - i am fine to remove it

I do believe it's optional; these are additional parameters that they offer for software referencing. If there is a plan to make a paper for a journal, conference, etc. especially in something like [Journal of Open Source Software](https://joss.theoj.org/), these optional parameters (DOI, SW, URL and other) would come into play (serving for better referencing).